src/views/microbit/l10n-static.json

```diff
@@ -0,0 +1,3 @@
+{
+    "cards.microbit-cardsLink": "https://resources.scratch.mit.edu/www/cards/en/microbit-cards.pdf"
```

Comment:
Yes, we can't just add a link in the l10n file because a translator doesn't know whether a translated file is available, or how to reference it. The resources server generates `localized-urls.json` based on what's actually available on the server.

Reply:
Thanks. So does this look good now?